.github/workflows/test.yml

```diff
@@ -231,6 +231,9 @@ jobs:
         run: |
           make mockgen -j $(nproc)
           hack/tree_status.sh
+      - name: spoof crun on arm64
+        if: ${{ matrix.run.arch == "arm64" }}
+        run: cp $(which runc) /usr/bin/crun
```

Comment:
Hm, which tests require the binary to be existent? Could we mock them as executable `os.Tempfile` to be system independent.

Reply:
I agree, I think that could be a follow up. I could open an issue if this merges (only because it's been such a pain to get tests passing here)

Reply:
Yes, maybe more general: All unit tests should be independent from external commands.

Reply:
agreed!

Reply:
This could have been a symbolic link, but a `cp` if fine, too. :smile: